packages/odie-client/src/components/message/dislike-feedback-message.tsx

```diff
@@ -55,7 +55,7 @@ export const DislikeFeedbackMessage = () => {
 						} }
 					>
 						{ __(
-							'I’m sorry my last response didn’t meet your expectations! Here’s some other ways to get more in-depth help:',
+							'Let’s get the information you need. Would you like to contact our support team?',
```

Comment:
@escapemanuele , @lucasmendes-design 
`support` is not capitalized here, but below in `ODIE_ERROR_MESSAGE`, `support` is capitalized. Is this intentional?